manager/controllers/mockup/policymanager.go

```diff
@@ -42,28 +42,69 @@ func (m *MockPolicyManager) GetPoliciesDecisions(input *policymanager.GetPolicyD
 		// no need to construct any result item
 	case "deny-dataset":
 		actionOnDataset := taxonomy.Action{}
-		(&actionOnDataset).Name = "Deny"
+		action := make(map[string]interface{})
+		action["name"] = "Deny"
+		denyAction := map[string]interface{}{}
+		action["Deny"] = denyAction
+
+		actionBytes, errJSON := json.MarshalIndent(action, "", "\t")
+		if errJSON != nil {
+			return nil, fmt.Errorf("error Marshalling External Catalog Connector Response: %v", errJSON)
+		}
+		err := json.Unmarshal(actionBytes, &actionOnDataset)
+		if err != nil {
+			return nil, fmt.Errorf("error in unmarshalling actionBytes : %v", err)
+		}
+
```

Comment:
1. Does taxonomy require additional properties for Deny action?
2. This code is duplicated in two more places.

Reply:
1. As of now, no. Please check https://github.com/fybrik/fybrik/blob/67d8886ea0bf58e80c2b1a6999f53c20d6bf2998/samples/taxonomy/example/module/actions.yaml#L27
2. Fixed code duplication. Please check if ok. Thanks.